cmd/subctl/verify.go

```diff
@@ -0,0 +1,286 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subctl
+
+// nolint:revive // Blank imports below are intentional.
+import (
+	"errors"
+	"fmt"
+	"github.com/submariner-io/submariner-operator/internal/exit"
```

Comment:
This needs to be moved into the second set of imports.

Reply:
Done